tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -391,29 +395,36 @@ func restoreCommandUse(cmd *cobra.Command, origionCommandUse string) {
 	cmd.Use = origionCommandUse
 }
 
+func redirectReomveSchedulerToDelteConfig(cmd *cobra.Command, schedulerName string, args []string) {
```

Comment:
Is `redirectReomveSchedulerToDelteConfig` a typo of `redirectReomveSchedulerToDeleteConfig`?